tests/e2e/default/enketo/extension-lib-form.wdio-spec.js

```diff
@@ -16,6 +16,11 @@ describe('Extension lib xpath function', () => {
     await commonPage.reloadSession();
   });
 
+  after(async () => {
+    await utils.deleteDocs(['form:extension-lib-average-calculator']);
+    await utils.revertDb([/^form:/], true);
```

Comment:
I believe that `utils.revertDb` is already called in the generic `after` defined here: https://github.com/medic/cht-core/blob/master/tests/wdio.conf.js#L336 SO this should not be be necessary.